src/circuit/circuit_vanilla.rs

```diff
@@ -858,28 +861,30 @@ mod tests {
             .unwrap();
 
         let circuit = OrchardCircuitVanilla {
-            path: Value::unknown(),
-            pos: Value::unknown(),
-            g_d_old: Value::unknown(),
-            pk_d_old: Value::unknown(),
-            v_old: Value::unknown(),
-            rho_old: Value::unknown(),
-            psi_old: Value::unknown(),
-            rcm_old: Value::unknown(),
-            cm_old: Value::unknown(),
-            psi_nf: Value::unknown(),
-            alpha: Value::unknown(),
-            ak: Value::unknown(),
-            nk: Value::unknown(),
-            rivk: Value::unknown(),
-            g_d_new: Value::unknown(),
-            pk_d_new: Value::unknown(),
-            v_new: Value::unknown(),
-            psi_new: Value::unknown(),
-            rcm_new: Value::unknown(),
-            rcv: Value::unknown(),
-            asset: Value::unknown(),
-            split_flag: Value::unknown(),
+            witnesses: Witnesses {
+                path: Value::unknown(),
+                pos: Value::unknown(),
+                g_d_old: Value::unknown(),
+                pk_d_old: Value::unknown(),
+                v_old: Value::unknown(),
+                rho_old: Value::unknown(),
+                psi_old: Value::unknown(),
+                rcm_old: Value::unknown(),
+                cm_old: Value::unknown(),
+                psi_nf: Value::unknown(),
+                alpha: Value::unknown(),
+                ak: Value::unknown(),
+                nk: Value::unknown(),
+                rivk: Value::unknown(),
+                g_d_new: Value::unknown(),
+                pk_d_new: Value::unknown(),
+                v_new: Value::unknown(),
+                psi_new: Value::unknown(),
+                rcm_new: Value::unknown(),
+                rcv: Value::unknown(),
+                asset: Value::unknown(),
+                split_flag: Value::unknown(),
+            },
```

Comment:
This should be replaced with
`witnesses: Witnesses::default()` or similar (all occurrences)
(need to make sure `default()` properly defined.)